src/opera/test/data/test_rtc_s1_config.yaml

```diff
@@ -0,0 +1,174 @@
+RunConfig:
+    Name: OPERA-RTC-S1-PGE-TEST-CONFIG
+
+    Groups:
+        PGE:
+            PGENameGroup:
+                PGEName: RTC_S1_PGE
+
+            InputFilesGroup:
+                InputFilePaths:
+                    - rtc_s1_test/input_dir/SAFE.zip
+                    - rtc_s1_test/input_dir/ORBIT.EOF
+
+            DynamicAncillaryFilesGroup:
+                AncillaryFileMap:
+                    dem_file: rtc_s1_test/input_dir/dem.tif
+
+            ProductPathGroup:
+                OutputProductPath: rtc_s1_test/output_dir
+                ScratchPath: rtc_s1_test/scratch_dir
+
+            PrimaryExecutable:
+                ProductIdentifier: RTC_S1
+                ProgramPath: mkdir
+                ProgramOptions:
+                    - '-p rtc_s1_test/output_dir/t069_147170_iw1/;'
+                    - '/bin/echo hello world > rtc_s1_test/output_dir/t069_147170_iw1/rtc_product.nc;'
+                    - '/bin/echo RTC-S1 invoked with RunConfig'
+                ErrorCodeBase: 300000
+                SchemaPath: pge/rtc_s1/schema/rtc_s1_sas_schema.yaml
+                IsoTemplatePath:
```

Comment:
Similiarly, will we have CompositeReleaseID (not sure if we want to add it before ticket 149 is merged but we should remember these additions if they apply.